docs/src/main/sphinx/sql/set-path.rst

```diff
@@ -0,0 +1,54 @@
+========
+SET PATH
+========
+
+Synopsis
+--------
+
+.. code-block:: text
+
+    SET PATH path-element[, ...]
+
+Description
+-----------
+
+Define a collection of paths to functions or table functions in specific
+catalogs and schemas for the current session.
+
+Each path-element uses a period-separated syntax of to specify the catalog name
+and schema location ``<catalog>.<schema>``of the function or only the schema
+location ``<schema>`` in the current catalog. The current catalog is set with
+:doc:`use` or as part of a client tool connection. Catalog and schema must
+exist.
+
+The defined path is used in addition to the global, built-in path.
```

Comment:
What does this mean?

Reply:
This is based on the input from @kasiafi that `global.builtin` is always tried first. So I guess this is like operating system/unix path stuff where you have a bunch of things on the path out of the box and you can add more. Should we somehow reword ? Or remove this?

Reply:
Should I remove this @martint ?

Reply:
Done